python/pyspark/ml/clustering.py

```diff
@@ -69,6 +70,25 @@ class KMeans(JavaEstimator, HasFeaturesCol, HasPredictionCol, HasMaxIter, HasTol
     True
     >>> rows[2].prediction == rows[3].prediction
     True
+    >>> import os, tempfile
```

Comment:
hmm... Here we combine the test and example functions. I do not have strong preference about whether this should live here or in tests file. @jkbradley